app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1><%= "#{@recipe.name}" %></h1>
+  <%= "#{@recipe.name}" %>
+  <p><%= "Complexity: #{@recipe.complexity} out of 10" %></p>
+  <p><%= "Genre: #{@recipe.genre}" %></p>
+<h3>Ingredients Needed:</h3>
+  <% @ingredients.each do |ingredient| %>
+    <p><%= ingredient.name %></p>
+  <% end %>
+<p>Total Cost: <%= @total_cost%><p/>
```

Comment:
Here is where instead of creating
```ruby
@total_cost = @ingredients.sum(:cost)
```
in your controller, you would have a method called `total_cost` in your `recipe` model and then you would just call `@recipe.total_cost` here.